docs/framework/interop/default-marshaling-for-strings.md

```diff
@@ -104,6 +105,8 @@ class StringLibAPI
     [DllImport("StringLib.dll")]
     public static extern void PassLPTStr([MarshalAs(UnmanagedType.LPTStr)] string s);
     [DllImport("StringLib.dll")]
+    public static extern void PassLPUTF8Str([MarshalAs(UnmanagedType.LPUTF8Str)] string s);
```

Comment:
Do we need so many examples?

Reply:
I think that the number of examples is appropriate, @NextTurn.